OracleDatabase/dockerfiles/12.1.0.2/installPerl.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+
+# Install latest Perl
+cd $INSTALL_DIR
+mv $ORACLE_HOME/perl $ORACLE_HOME/perl.old
+wget http://www.cpan.org/src/5.0/perl-5.14.1.tar.gz
```

Comment:
CPAN doesn't seem to support HTTPS. Will a checksum be enough or do we need GPG keys?